src/viur_cli/local.py

```diff
@@ -173,6 +173,19 @@ def check(dev):
         utils.echo_info("\U00002714 No vulnerabilities found.")
 
 
+@cli.command()
+@click.option("--directory", "-d", default="deploy")
+def excheck(directory):
+    for root, _, files in os.walk(directory):
+        for file in files:
+            if file.endswith(".py"):
```

Comment:
`os.walk` is old-fashion stuff. The two `for` loops and the `if` can be simlified by `for file in pathlib.Path(directory).rglob("*.py"):`